dcpy/utils/geospatial.py

```diff
@@ -12,6 +15,21 @@
 
 from dcpy.utils.logging import logger
 
+GEOMETRY_FORMATS = {
+    "WKT": "EPSG:4326",
+    "WKB": "ESPG:3867",
```

Comment:
Not entirely sure how this is going to be used, but is there a reason to map a format to a specific projection?

Reply:
good call, this should just be a list or enum. and I think I'll rename some things to not imply all coordinate reference systems are proejcted

interesting background on ESPG stuff I just stumbled through:
a European Petroleum Survey Group (EPSG) code is just a unique code that can represent many types of things like coordinate systems, datums, and spheroids ([source](https://support.virtual-surveyor.com/support/solutions/articles/1000261353-what-is-an-epsg-code-)).

we have a few coordinate systems of interest. some are geodesic like [EPSG:4326](https://epsg.io/4326) and have units of degrees. some are projected like [EPSG:4087](https://epsg.io/4087) for the world in meters and [EPSG:2236](https://epsg.io/2263) for NY in meters